src/readPart10Header.js

```diff
@@ -15,31 +15,51 @@ import readDicomElementExplicit from './readDicomElementExplicit.js';
  *         elements successfully parsed before the error.
  */
 
-export default function readPart10Header (byteArray, options) {
+export default function readPart10Header(byteArray, options) {
   if (byteArray === undefined) {
     throw 'dicomParser.readPart10Header: missing required parameter \'byteArray\'';
   }
 
   const littleEndianByteStream = new ByteStream(littleEndianByteArrayParser, byteArray);
 
-  function readPrefix () {
+  function readPrefix() {
+    const { TransferSyntaxUID } = options || {};
```

Comment:
options belongs to scope parent of this one. 
If possible can you do one of the options.

1. readPart10Header(byteArray, options={}) {

2. function readPrefix() {
    const { TransferSyntaxUID } = options || {};
    
    Just once because the TransferSyntaxUID is being shadowed

Reply:
Thank you - good idea.